bundle/manifests/sonataflow-operator.clusterserviceversion.yaml

```diff
@@ -537,4 +537,4 @@ spec:
   minKubeVersion: 1.23.0
   provider:
     name: Red Hat
-  version: latest
+  version: 2.0.0-snapshot
```

Comment:
Shouldn't we revert it back for now to `latest`? Still not sure. Or just merge as is.

Reply:
We need to fix the CI :/

I think @davidesalerno already opened a JIRA.